(node:873813) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-session-46151',
  containerIp: '10.0.3.61',
  mitmPort: 46151,
  autoAccessFixed: 1
}
2024-11-19 09:16:06.610 - [Info] MITM Version: 2
2024-11-19 09:16:06.612 - [Info] Auto Access Enabled: true
2024-11-19 09:16:06.613 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-19 09:16:06.636 - [Info] SSH man-in-the-middle server for 10.0.3.61 listening on 10.0.3.1:46151
2024-11-19 09:17:28.986 - [Debug] [Connection] Attacker connected: 34.41.17.26 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-19 09:17:29.681 - [Debug] [Auth] Attacker 34.41.17.26 trying to authenticate with 'password'
2024-11-19 09:17:29.683 - [Debug] [Auto Access] Attacker: 34.41.17.26, Threshold: 1, Attempts: 1
2024-11-19 09:17:29.683 - [Debug] [Auto Access] Compromising the honeypot
2024-11-19 09:17:29.683 - [Debug] [Auto Access] Adding the following credentials: 'deploy:passw0rd123'
2024-11-19 09:17:30.022 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-19 09:17:30.025 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-19 09:17:30.026 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.61
2024-11-19 09:17:30.123 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-19 09:17:30.461 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-19 09:17:30.964 - [Debug] [Connection] Attacker connected: 34.41.17.26 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-19 09:17:31.431 - [Debug] [Auth] Attacker 34.41.17.26 trying to authenticate with 'password'
2024-11-19 09:17:31.432 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-19 09:17:31.433 - [Debug] [Auth] Attacker: 34.41.17.26 has so far made 1 attempts. Remaining: 5 attempts
2024-11-19 09:17:31.539 - [Debug] [Connection] Attacker closed the connection
2024-11-19 09:17:31.687 - [Debug] [Connection] Attacker connected: 34.41.17.26 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-19 09:17:32.243 - [Debug] [Auth] Attacker 34.41.17.26 trying to authenticate with 'password'
2024-11-19 09:17:32.244 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-19 09:17:32.245 - [Debug] [Auth] Attacker: 34.41.17.26 has so far made 1 attempts. Remaining: 5 attempts
2024-11-19 09:17:32.459 - [Debug] [Connection] Attacker closed the connection
2024-11-19 09:17:32.514 - [Debug] [Connection] Attacker closed connection
2024-11-19 09:17:32.529 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-19 09:17:32.531 - [Debug] [LXC] Container's OpenSSH server closed connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
